src/platform/posix.c

```diff
@@ -757,4 +758,31 @@ uint8 platform_get_locale_temperature_format(){
 	return TEMPERATURE_FORMAT_C;
 }
 
+bool platform_get_font_path(TTFFontDescriptor *font, utf8 *buffer)
+{
```

Comment:
The other functions in this file has the `{` on the same row as the function declaration